syntax = "proto3";
package task;





/*
ListInspectionInfo请求
*/
message ListInspectionInfoRequest {
    /*
    套件id
    */
    string pluginId = 1 ;
    /*
    页码
    */
    int32 page = 2 ;
    /*
    每页大小
    */
    int32 pageSize = 3 ;
}

/*
ListInspectionInfo返回
*/
message ListInspectionInfoResponse {
message List {
    /*
    巡检对象个数
    */
    int32 num = 1 ;
    /*
    巡检任务Id
    */
    string inspectionTaskId = 2 ;
    /*
    巡检任务名称
    */
    string name = 3 ;
    /*
    报告模板ID
    */
    string templateId = 4 ;
    /*
    报告模板名
    */
    string templateName = 5 ;
    /*
    备注
    */
    string memo = 6 ;
    /*
    巡检策略(once单次,crontab周期)
    */
    string taskType = 7 ;
}
        
    
    /*
    页数
    */
    int32 page = 1 ;
    /*
    页大小
    */
    int32 page_size = 2 ;
    /*
    总数
    */
    int32 total = 3 ;
    /*
    数据列表
    */
    repeated List list = 4 ;
}

/*
ListInspectionInfoApi返回
*/
message ListInspectionInfoResponseWrapper {
    /*
    返回码
    */
    int32 code = 1 ;
    /*
    返回码解释
    */
    string codeExplain = 2 ;
    /*
    错误详情
    */
    string error = 3 ;
    /*
    返回数据
    */
    task.ListInspectionInfoResponse data = 4 ;
}

